staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -181,6 +181,10 @@ type VolumeSource struct {
 	//
 	// +optional
 	Ephemeral *EphemeralVolumeSource `json:"ephemeral,omitempty" protobuf:"bytes,29,opt,name=ephemeral"`
+	// oci represents a OCI object (image or artifact) pulled and mounted on the kubelet's host machine.
```

Comment:
Godoc should include:

* Describe when the volume is resolved (at pod startup) and describe how the policy impacts that resolution.
* Describe whether the volume is reresolved.
* Describe that the OCI object will be mounted as a directory.
* Describe that the volume will be read only.

Reply:
Added all the docs as requested.